terraform/modules/application/parameters.tf

```diff
@@ -0,0 +1,10 @@
+data "aws_ssm_parameter" "partial_database_connection_string" {
+  name = var.database_partial_connection_string_arn
+}
+
+resource "aws_ssm_parameter" "complete_database_connection_string" {
+  #checkov:skip=CKV_AWS_337:default encryption not using a kms cmk sufficient
+  name  = "COMPLETE_DATABASE_CONNECTION_STRING"

```

Comment:
This name may need to have the prefix in, so that it's unique across different app instances.